templates/github/.ci/ansible/ssh/id_ed25519

```diff
@@ -0,0 +1,7 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
we will probably need a [.gitleaks.toml](https://github.com/ansible/galaxy_ng/blob/master/.gitleaks.toml)